Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Qualys_Enrichment.yml

```
@@ -612,14 +612,104 @@ tasks:
           root: Qualys.Endpoint
           accessor: ID
       val3:
-        simple: N/A
+        simple: n/a
     separatecontext: false
     continueonerrortype: ""
     view: |-
       {
         "position": {
-          "x": 320,
-          "y": 1210
+          "x": 690,
+          "y": 1360
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "17":
+    id: "17"
+    taskid: c737255d-4571-477a-8887-b29ea923efc1
+    type: regular
+    task:
+      id: c737255d-4571-477a-8887-b29ea923efc1
+      version: -1
+      name: Set ASM - Enrichment Status
```

```suggestion
      name: Set ASM enrichment status to true
```
Consistency with other playbooks